tgm/hooks/hooks.py

```diff
@@ -58,6 +58,10 @@ def __call__(self, dg: DGraph, batch: DGBatch) -> DGBatch:
         if hasattr(batch, 'nbr_nids'):
             for hop in range(len(batch.nbr_nids)):
                 nids.append(batch.nbr_nids[hop].to(batch.src.device))
+        if hasattr(batch, 'node_ids'):
+            nids.append(
+                batch.node_ids.to(batch.src.device)
+            ) if batch.node_ids is not None else None
```

Comment:
```suggestion
        nids.append(
            batch.node_ids.to(batch.src.device)
        ) if batch.node_ids is not None else None
```

since `batch.node_ids` always exists as an attribute